File: Python/car.py
class car:
    def carcommand(self):
        command=0
        enter=input("Do you want to enter in the car ")
        while enter=="yes":
            com=input(" ENTER: START/STOP ")
            if com=="start":
                 if command==1:
                     print("Car already Started")
                 else:
                     print("Car  Started")
                     command=1
            elif com=="stop":
                    if command==0:
                        print("Car already stopped")
                    else:
                        command=0
                        print("car stopped")
            elif com=="exit":
                break
        
            else:
                 print("Invalid command")
        else:
             print("Invalid command")
obj=car()
obj.carcommand()

File: C#/car.cs
using System;
class car
{
    public static void Main()
    {
        int command=0;
        string choice="yes";
        while(choice=="yes")
        {
         Console.WriteLine("Enter the command");
         string com=Console.ReadLine();
         if(com=="start")
        {
         if(command==1)
         {
             Console.WriteLine("Car already Started");
             
         }
         else
         {
         Console.WriteLine("Car  Started");
         command=1;
         }
         
        }
        else if (com=="stop")
        {
            if(command==0)
            {
            Console.WriteLine("Car already stopped");
            }
            else
            {
                command=0;
                Console.WriteLine("car stopped");
            }
        }
        else if (com=="exit")
        {
            break;
        }
        else
        {
            Console.WriteLine("Invalid command");
        }
        }
    }

}